src/pages/library/Showcase.js

```diff
@@ -231,25 +52,7 @@ const Showcase = () => {
         lowerName: _.toLower(workspace.name), lowerDescription: _.toLower(workspace.description)
       }), showcase)
 
-      const workspacesByTag = _.omitBy(_.isEmpty, groupByFeaturedTags(featuredWorkspaces))
-
-      // Trim items from the sidebar for which there aren't any featured workspaces.
-      const activeTags = _.keys(workspacesByTag)
-      const activeSections = _.flow(
-        _.map(section => {
-          const activeLabels = _.intersectionBy(_.toLower, section.labels, activeTags)
-          return {
-            ...section,
-            labels: activeLabels,
-            tags: _.map(_.toLower, activeLabels)
-          }
-        }),
-        _.remove(section => _.isEmpty(section.labels))
-      )(sidebarSections)
-
       setFeaturedList(featuredWorkspaces)
-      setWorkspacesByTag(workspacesByTag)
-      setSections(activeSections)
 
       // TODO: is this actually doing anything for us?
```

Comment:
Since you're touching this, would you mind removing this TODO? If you're so inclined, replace it with a comment about what this does (see the video I added to the previous PR).